### PR TITLE
Add docs for MFA feature

### PR DESCRIPTION
**Pull request by Roma36**
# Description of the change

This PR adds documentation for MFA feature.

## Checklist

Please make sure that the proposed change checks all the boxes below before requesting a review:

- [x] I have reviewed the [guidelines for contributing](https://github.com/spacelift-io/user-documentation/blob/main/CONTRIBUTING.md) to this repository.
- [x] The preview looks fine.
- [x] The tests pass.
- [x] The commit history is clean and meaningful.
- [x] The pull request is opened against the `main` branch.
- [x] The pull request is no longer marked as a draft.
- [x] You agree to license your contribution under the [MIT license](https://github.com/spacelift-io/user-documentation/blob/main/LICENSE) to Spacelift (not required for Spacelift employees).
- You have updated the navigation files correctly:
    - [ ] No new pages have been added, or;
    - [ ] Only _nav.yaml_ has been updated because the changes only apply to SaaS, or;
    - [ ] Only _nav.self-hosted.yaml_ has been updated because the changes only apply to Self-Hosted, or;
    - [x] Both _nav.yaml_ and _nav.self-hosted.yaml_ have been updated.

If the proposed change is ready to be merged, please request a review from `@spacelift-io/solutions-engineering`. Someone will review and merge the pull request.

_Spacelift employees should request reviews from the relevant engineers and are allowed to merge pull requests after they got at least one approval._

Thank you for your contribution! 🙇

### CODE REVIEW

File: docs/product/security/mfa.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+## Setting Up MFA for Your Account
+
+### Enable MFA
+
+Go to the _Personal settings_. Next, navigate to _Multi-factor authentication_. Here, you can add personal security keys that will be used for authentication.
+
+### Adding Security Keys
+
+- Click on the _Enable_ button to activate MFA.
+![](./personal-settings-enable-mfa.png)
```
Comment (jmfontaine) [inline docs/product/security/mfa.md:17]: Screenshots must be stored in the `docs/assets/screenshots` folder.

File: docs/product/security/mfa.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+You can remove a security key at any time. To delete, click the trash icon next to the key you wish to remove and confirm your action.
+
+## Setting Up MFA for Organization
+
+In order to manage MFA in your organization, please go to the _Organization settings_. Next, navigate to _Multi-factor authentication_.
+
+Admins can view and delete security keys for any user within the organization to maintain the organization's security integrity.
+
+![](./organization-settings-mfa.png)
```
Comment (jmfontaine) [inline docs/product/security/mfa.md:31]: Screenshots must be stored in the `docs/assets/screenshots` folder.

**Review by jmfontaine — DISMISSED** [review]
Except for the misplaced screenshots, it looks good to me.

**Comment by Roma36** [comment]
@jmfontaine I have addressed your comments and rebased this branch, but for some reason, one external link is failing check: 
https://git.alpinelinux.org/aports/tree/testing/spacectl/APKBUILD

**Review by michieldewilde — APPROVED** [review]

**Review by jmfontaine — APPROVED** [review]
Looks good to me.

**Review by KNiepok — APPROVED** [review]

**Pull request merged**